File: app/vn/vnext/sefuri/sf/json/request/SF0030211Req.java
package vn.vnext.sefuri.sf.json.request;

import com.fasterxml.jackson.annotation.JsonProperty;
import vn.vnext.sefuri.sf.json.common.AbstractJson;
import vn.vnext.sefuri.sf.json.core.OfferJson;

/**
 * Created by TungNT on 1/5/2017.
 */
public class SF0030211Req extends AbstractJson {
    @JsonProperty("offer")
    private OfferJson offerJson;

    public OfferJson getOfferJson() {
        return offerJson;
    }

    public void setOfferJson(OfferJson offerJson) {
        this.offerJson = offerJson;
    }
}

File: client/src/app/request/SF0030211.req.ts
import {Offer} from "../model/core/Offer.model";
export class SF0030211Req {
    offer: Offer;
    dealCode: string;
    productCode: string;

    constructor(offer: Offer, dealCode: string,productCode: string) {
        this.offer = offer;
        this.dealCode = dealCode;
        this.productCode = productCode;
    }
}
